ruuvitag_sensor/__init__.py

```diff
@@ -1 +1,6 @@
-__version__ = '2.0.0'
+try:
+    import importlib.metadata  # >=3.8
```

Comment:
What does it mean if a "package is no longer provisional"? [Reference](https://docs.python.org/3/library/importlib.metadata.html#:~:text=Changed%20in%20version%203.10%3A%20importlib.metadata%20is%20no%20longer%20provisional.)

Reply:
Earlier that module was temporary (or not final), so there was a possibility that it would have still changed.
https://docs.python.org/3.8/library/importlib.metadata.html